Automattic/jetpack
# Search: add @wordpress/babel-preset-default package

Broken out from https://github.com/Automattic/jetpack/pull/20369, where package.json is causing rebase headaches. Seems sensible to land this separately.

#### Changes proposed in this Pull Request:

* Add @wordpress/babel-preset-default package to packages in the Jetpack plugin. This will be used by the new search customization UI in wp-admin.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Ensure that you can build search with `pnpm build && pnpm watch-search`.


cc @jsnmoon
